projects/storefrontstyles/scss/components/cart/_cart-coupon.scss

```diff
@@ -87,4 +90,14 @@
       text-decoration: underline;
     }
   }
+
+  cx-applied-coupons {
+    .row {
+      width: calc(100% + 15px);
```

Comment:
**Can you explain why +15px 🤔 , and where that number pops out from?** 

We don't need to pollute this stylesheet as this wasn't the root cause
Their exist `projects/storefrontstyles/scss/components/cart/_applied-coupons.scss`. 

I'm assuming you're trying to fix the extra width that comes with that coupon, but we shouldn't hotfix it and we solve the issue at hand

After some investigation, https://github.com/SAP/spartacus/blob/develop/projects/storefrontstyles/scss/components/cart/_applied-coupons.scss#L42 is the issue. Removing it works, and RTL is still supported

Reply:
No, this value is related to `row` class which is from bootstrap. There are style with (minus) -15px for margins for the row.

Reply:
I wanted to write that this is because of row classes from bootstrap and -15px for margins in styles. But you are genius :) and your solutions works! :)